sensors/ev3_uart_sensor_ld.c

```diff
@@ -232,12 +232,12 @@ struct ev3_uart_port_data {
 	u8 requested_mode;
 	u8 type_id;
 	u8 new_mode;
-	u32 raw_min;
-	u32 raw_max;
+	s32 raw_min;
+	s32 raw_max;
 	u32 pct_min;
```

Comment:
Yes drop this commit. The shared struct is in <https://github.com/ev3dev/lego-linux-drivers/blob/master/include/lego_sensor_class.h#L67>, and already uses signed values.